### PR TITLE
Checklist and coordination: Major upgrade to v3.0.0

### PR DESCRIPTION
**Pull request by sagarpreet-chadha**
1. Bootstrap major version upgrade (noting  Merged `Bump bootstrap from 4.3.1 to 4.5.3` #645)

## To do:

* [ ] complete this PR
* [ ] open a matching PR on `plots2` for this new version # 

* [x] open an issue using the "release" template with this checklist with title `Checklist and coordination for v0.0.0 major/minor/patch release` (see [semantic versioning](https://docs.npmjs.com/about-semantic-versioning/))
* [ ] compile release notes below from corresponding [release milestone](https://github.com/publiclab/PublicLab.Editor/milestones)
* [ ] update version number in `package.json` (ex #1695)
* [ ] finalize and merge to `main` branch (freeze merges to `main` branch until next step)
* [ ] merge, build and publish `/dist/` files to `stable` (merges to `main` branch can resume for next release)
* [ ] create a release on GitHub and use features description + release notes from below
* [ ] tag version number branch (i.e. `v3.0.0`)
* [ ] publish tagged branch to `npm`
* [ ] publish to live `github-pages` demo (with [bash script](https://github.com/publiclab/image-sequencer/pull/1703) from `/scripts/update_demo`) (from within GitPod works well)
* [ ] close this issue!

### CODE REVIEW

**Comment by sagarpreet-chadha** [comment]
@jywarren let's publish this :)

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "publiclab-editor",
-  "version": "2.2.0",
+  "version": "3.2.0",
```
Comment (jywarren) [inline package.json:3]:
```suggestion
  "version": "3.0.0",
```

Here we'd zero out the minor and patch numbers if we are doing a major. Is it 100% required though? Does our code break if you have a Bootstrap 3 template? Just checking!

**Review by jywarren — CHANGES_REQUESTED** [review]
At image-sequencer we've been refining a release checklist workflow which is quite strict. Could be interesting to see how we are doing the coordination leading up to a new release! publiclab/image-sequencer#1751 - it's actually stored as an issue template so we open a new issue for each, and go through the checklist.

Here, for example, one checklist item might be to create a corresponding issue in `plots2` - also, if the checklist is an issue, then it can last after the PR is closed, for tasks that come after the PR. What do you think? No big deal for this one but just an idea.

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "publiclab-editor",
-  "version": "2.2.0",
+  "version": "3.0.0",
```
Comment (sagarpreet-chadha) [inline package.json:3]: Oops I mistakenly committed `3.2.0` 🙈

File: package.json (hunk 1 above)
Comment (sagarpreet-chadha) [reply package.json]: Yes @jywarren  this is a breaking change and CSS will break for people on lower bootstrap version.

**Comment by sagarpreet-chadha** [comment]
Sure, we can use the checklist this time itself 👍

**Comment by jywarren** [comment]
Oh cool! Well it's a huge checklist and we don't need to do every step as we do in IS - but great! OK, this should be good then?

**Review by jywarren — APPROVED** [review]

**Comment by sagarpreet-chadha** [comment]
Yes this is good to go 🚀

**Comment by jywarren** [comment]
Great! Let's make a github release too, just because we're calling this a major?

**Comment by jywarren** [comment]
I'll publish from the main branch to NPM, too!

**Comment by sagarpreet-chadha** [comment]
https://github.com/publiclab/PublicLab.Editor/releases/tag/v3.0

Already done using the create-release script 🙈

**Comment by jywarren** [comment]
Just noting:

> ah, we skipped the build dist files step (thinking of NPM) but there haven't been any functional changes to that file since the last time it was built, so we should be OK: https://github.com/publiclab/PublicLab.Editor/commits/main
the dependabot updates may not all be baked in for upstream deps we compile in, but it's probably fine

> indeed, nothing would have been compiled in now that I look at the specific commits. Browserify was updated but that's probably fine too.

**Comment by jywarren** [comment]
Published as `v3.0.0`!!! :tada:

https://www.npmjs.com/package/publiclab-editor

**Comment by sagarpreet-chadha** [comment]
Awesomeee 🎉

**Pull request merged**